test/itemlist_formaction.cpp

```diff
@@ -56,35 +55,35 @@ TEST_CASE("process_op(OP_OPEN)", "[itemlist_formaction]") {
 	REQUIRE_NOTHROW(itemlist.process_op(OP_OPEN));
 
 	std::ifstream pagerFileStream (pagerfile.getPath());
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == pager_prefix_title + test_title);
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == pager_prefix_author + test_author);
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == pager_prefix_date + test_pubDate_str);
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == pager_prefix_link + test_url);
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == " ");
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == test_description);
 
-	REQUIRE( std::getline (pagerFileStream,line) );
+	REQUIRE(std::getline (pagerFileStream,line));
 	REQUIRE(line == "");
 }
 
-TEST_CASE("process_op(OP_DELETE)", "[itemlist_formaction]") {
+TEST_CASE("OP_DELETE", "[itemlist_formaction]") {
 	//REQUIRE_NOTHROW(itemlist.process_op(OP_DELETE));
 	//REQUIRE(feed->total_item_count() == itemCount -1);
 	//Crash, to investigate
 }
 
-TEST_CASE("process_op(OP_PURGE_DELETED)", "[itemlist_formaction]") {
+TEST_CASE("OP_PURGE_DELETED works properly", "[itemlist_formaction]") {
```

Comment:
This is of no immediate consequence, as the test is empty at the moment, but a note for the future: I stopped using words "properly", "correctly", "as expected" and so on because they don't explain *what* behaviour is expected.